src/components/profile-view/user-update.jsx

```diff
@@ -0,0 +1,137 @@
+import React, {useState} from 'react';
+import axios from 'axios';
+import { Form, Button } from 'react-bootstrap';
+
+export function UserUpdate (props) {
+  const {onUserUpdate} = props;
+
+  const [ username, setUsername ] = useState('');
+  const [ password, setPassword ] = useState('');
+  const [ email, setEmail ] = useState('');
+  const [ birthday, setBirthday ] = useState('');
+
+  const [usernameErr, setUsernameErr]=useState('');
+  const [passwordErr, setPasswordErr]=useState('');
+  const [emailErr, setEmailErr]=useState('');
+
+  const token=localStorage.getItem('token');
+  const user=localStorage.getItem('user');
+
+  function validate(){
+    let isReq=true;
+
+    if(!username) {
+      setUsernameErr('Username is required.');
+      isReq=false;
+    } else if (username.length < 5) {
+      setUsernameErr('Username must be at lease 5 characters long.');
+      isReq=false;
+    }
+
+    if(!password) {
+      setPasswordErr('Password is required.');
+      isReq=false;
+    } else if (password.length < 6) {
+      setPasswordErr('Password must be at lease 6 characters long.');
+      isReq=false;
+    }
+
+    if(!email) {
+      setEmailErr('Email address is required.');
+      isReq=false;
+    } else if (email.indexOf('@') === -1) {
+      setEmailErr('Email address is not valid.');
+      isReq=false;
+    }
+
+    return isReq;
+  }
+
+  function handleSubmit (e) {
+    e.preventDefault();
+    const isReq = validate();
+    let config={
+      headers: { Authorization: `Bearer ${token}`}
+    }
+    if (isReq) {
+      axios.put(`https://favmovie123.herokuapp.com/users/${user}`, {
+        Username: username,
+        Password: password,
+        Email: email,
+        Birthday: birthday
+      },config)
+      .then((response) => {
+        const data = response.data;
+        alert('Update successful!');
```

Comment:
instead of using alert you can use toast from bootstrap, here is an [example ](https://react-bootstrap.github.io/components/toasts/) on how to use it.

Thank you!